src/expense/controllers-expenses.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const expSv = require('./services-expenses.js');
+
+const getAll = (req, res) => {
+  res.statusCode = 200;
```

Comment:
If I'm not mistaken, 200 status will be by default

Reply:
\+
and use app.status(200), what is the reason to avoid this method?)